apps/fz_http/test/fz_http/devices_test.exs

```diff
@@ -177,7 +177,7 @@ defmodule FzHttp.DevicesTest do
 
     @invalid_endpoint_ipv4_attrs %{
       use_site_endpoint: false,
-      endpoint: "265.1.1.1"
+      endpoint: "-265.1.1.1"
```

Comment:
This is (almost) a legal FQDN now due to a new RFC, so just making this *extra* invalid.